packages/firestore/src/api/credentials.ts

```diff
@@ -214,6 +214,7 @@ export class FirebaseCredentialsProvider implements CredentialsProvider {
       // outstanding so the response is potentially for a previous user (which
       // user, we can't be sure).
       if (this.tokenCounter !== initialTokenCounter) {
+        this.forceRefresh = true;
```

Comment:
Is this change related to the fix at hand?

Reply:
No, this is meant for a different PR. Removed.